packages/position/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "dist/index.js",
   "scripts": {
     "prepare": "npm run clean && npm run build",
-    "clean": "rm -rf dist && mkdir dist",
-    "build": "babel src/index.js -o dist/index.js"
+    "clean": "rm -rf dist",
+    "build": "babel src -d dist"
```

Comment:
Here too

Reply:
Just trying to debug the docker build – might ditch it altogether